### ISSUE TITLE
Issue running example config 

### ISSUE THREAD

**Issue by nishadsabnis** [alphapapa/yequake#11]
Actually I faced the same issue with the example config too.

Here is my example config

```
   (setq yequake-frames
          '(("Yequake & scratch" .
             ((width . 0.75)
              (height . 0.5)
              (alpha . 0.95)
              (buffer-fns . ("~/Dropbox/org/projects.org"
                             split-window-horizontally
                             "*scratch*"))
              (frame-parameters . ((undecorated . t)))))))
```
I should note that this is Spacemacs running on MacOS High Sierra.

On `yequake-toggle` I get the error

```
Wrong type argument: listp, 0.75
```

**Comment by alphapapa** [comment]
As I said in the other issue, it works for me.  Something must be wrong on your end.  You'll have to provide a full backtrace with `debug-on-error` if you need help debugging it.

**Comment by nishadsabnis** [comment]
Here is some of the backtrace 
(The lower part is missing because I invoked this using counsel-M-x and the ivy-call is long as it contains all the interactive functions you can call) 
```
Debugger entered--Lisp error: (wrong-type-argument listp 0.75)
  yequake--toggle-frame("Projects" ((width . 0.75) (height . 0.5) (alpha . 0.95) (buffer-fns "~/Dropbox/org/projects.org") (frame-parameters (undecorated . t))))
  yequake-toggle("Projects")
  funcall-interactively(yequake-toggle "Projects")
  call-interactively(yequake-toggle record nil)
  command-execute(yequake-toggle record)
  #f(compiled-function (cmd) #<bytecode 0x41d6b1e1>)("yequake-toggle")

```

**Comment by alphapapa** [comment]
What version of Emacs?

**Comment by nishadsabnis** [comment]
Emacs 26.1. 
Also, I am running spacemacs on the latest develop branch. 
org-version is 9.2.1

Thanks for the help btw :)

**Comment by alphapapa** [comment]
I'm also using Emacs 26.1.  I don't understand how you could be getting that error, because it works fine for me using the code you provided.

I doubt that Spacemacs is causing it, but please try to reproduce with regular Emacs.  I can't provide help for Spacemacs.

If that still causes the problem, please try to reproduce with `emacs -q`.

I also doubt that MacOS is causing the problem, but since there are several different ways of acquiring Emacs on MacOS, and some of them build differently, it's possible that the one you are using has a bug in it.  How did you install Emacs?  If you haven't already tried using Homebrew to install it, you might consider that, because I've heard from users who use it successfully (in general, not with Yequake), however I can't vouch for Homebrew myself, and I've heard both positive and negative things about it.  If that doesn't work, you should probably try using the most plain, basic Emacs distribution for MacOS--directly from FSF, if possible.

The bottom line is that this is working for me and other users, using the same configuration code, so it's probably a problem on your end that only you can solve.  In case you discover a workaround, it might be possible to include it in this package.

**Comment by nishadsabnis** [comment]
Let me try some combination of stuff out and get back to you with my observations. Thanks.

**Comment by nishadsabnis** [comment]
Okay well its solved now though not sure why? Let me note down all the things that I changed. 
1. Updated to latest version of MacOS Mojave (also let me note that my emacs is installed via homebrew. I was meaning to do this anyways so just decided to pull the trigger. 

2. I went hunting on google and found this https://github.com/org2blog/org2blog/issues/216 which seemed to suggest I should try removing all compiled elisp files. I had previously tried removing compiled files in just the org-contrib diterctory but this time I used ```find . -name "*.elc" -type f | xargs rm -f``` in my .emacs.d directory.

3. Opened up Spacemacs and it works like a charm now! Looking forward to using it, this will improve my capture workflow for sure 😃. Thanks for your help, closing!

**Comment by alphapapa** [comment]
Stale byte-compiled `.elc` files are a common problem.  The best solution is to delete and reinstall the relevant packages.  If you just delete the elc files, they won't be byte-compiled anymore, and load times will increase.

**Comment by nishadsabnis** [comment]
Ahh okay, I was under the impression they got compiled again on the next load so thanks for mentioning this. So maybe I am not doing something right but with Spacemacs I did go through the update packages option. Guess it just did not clear out the old .elc files. Will look up how to delete packages properly in Spacemacs. 

I have been using emacs for a couple of years now but I must admit I am quite a beginner in terms of Lisp so as I learn more hopefully I make less of these mistakes and waste less of other people's time!

**Comment by alphapapa** [comment]
Spacemacs uses package.el AFAIK, but it adds an additional layer for configuring packages.

I've lost count of how many times I've had problems caused by stale .elc files.  I don't know why it happens, but sometimes it just does.

You may find this helpful for deleting old package versions: https://github.com/alphapapa/unpackaged.el#delete-all-installed-versions-of-a-package

**Comment by nishadsabnis** [comment]
Thanks, will check it out. Im now trying out org-super-agenda and org-sidebar! :)
